src/tools/cephfs/cephfs-shell

```diff
@@ -1551,15 +1556,44 @@ if __name__ == '__main__':
             args.commands = ['run_script ' +  args.batch, ',quit']
     if args.test:
         args.commands.extend(['-t,'] + [arg + ',' for arg in args.test])
+    if not args.batch and len(args.commands) > 0:
+        args.exe_and_quit = True
+
+    manage_sys_argv(args)
+
+    return args
 
+def manage_sys_argv(args):
+    exe = sys.argv[0]
     sys.argv.clear()
     sys.argv.append(exe)
     sys.argv.extend([i.strip() for i in ' '.join(args.commands).split(',')])
 
+def execute(args):
+    # launch shell else just execute command and exit shell session with the
+    # command's return value.
```

Comment:
```suggestion
'''
Launch a shell session if no commands are provided.
Otherwise commands are executed and session is quit with return value of last executed command.
'''
```